components/create-collective/CreateCollectiveForm.js

```diff
@@ -235,13 +255,15 @@ class CreateCollectiveForm extends React.Component {
                     >
                       {inputProps => (
                         <Field
+                          onClick={() => this.setState({ isURLFieldTouched: true })}
                           as={StyledInputGroup}
                           {...inputProps}
-                          prepend="opencollective.com"
+                          prepend="opencollective.com/"
                           placeholder={placeholders.slug}
                         />
                       )}
                     </StyledInputField>
+                    {values.name.length > 0 && !this.state.isURLFieldTouched && <SubText>Suggested</SubText>}
```

Comment:
`Suggested` should be internationalized. See https://docs.opencollective.com/help/contributing/development/translations